program/tests/tests/add_remove_validator.rs

```diff
@@ -108,7 +108,7 @@ async fn test_remove_validator_with_unclaimed_credits() {
     let vote_account = solido.validators.entries[0].pubkey;
     assert_eq!(
         solido.validators.entries[0].entry.fee_credit,
-        StLamports(62_301_530_769)
+        StLamports(998_758_598)
```

Comment:
Did you manage to find out what caused the difference? Is it a difference in the inflation schedule?

Reply:
No, I did't do a research yet. Just wanted do deliver ASAP